packages/manager/modules/overthebox/package.json

```diff
@@ -34,7 +34,7 @@
     "moment": "^2.24.0",
     "oclazyload": "^1.1.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
-    "ovh-api-services": "^9.42.0",
+    "ovh-api-services": "^9.43.0",
```

Comment:
Could you please rebase on `develop` branch?

To make sure that you will use the latest version of `ovh-api-services` from the monorepo.

```sh
$ grep -C 1 ovh-api-services@ yarn.lock

ovh-api-services@^9.44.0:
  version "9.44.0"
```